docs/operating-systems/course-tools.md

```diff
@@ -148,51 +143,52 @@ If you haven’t installed gatorgrade type this command:
 
 ### Go installation process:
 
-If you haven't installed go click the link and follow [the instructions](https://go.dev/dl/)
+If you haven't installed Go: [the instructions](https://go.dev/dl/)
 
 Make sure you click the right chip that your laptop has.
 
 If you choose the right version and it still isn't working, follow these instructions:
-1. type in the command ```nano ~/.zshrc``` into your terminal
```

Comment:
This formatting is still incorrect, in the deploy preview it is showing as a paragraph instead of the lines it appears like you are trying to format it as. Correct this and then this looks great!